setup.py

```diff
@@ -173,7 +173,7 @@ def load_description():
     entry_points={
         'console_scripts': [
             'falcon-bench = falcon.cmd.bench:main',
-            'falcon-print-routes = falcon.cmd.print_routes:main'
+            'falcon-inspect-app = falcon.cmd.inspect_app:main'
```

Comment:
I would prefer to keep the old alias for compatibility in case anyone is accustomed to that.

Reply:
I changed the alias since it does more than just logging the routes, but I think that we can keep the old alias or have both.